css/at-rules/font-face.json

```diff
@@ -43,6 +43,186 @@
+            }
+          }
+        },
+        "OpenType_COLRv1": {
+          "__compat": {
+            "description": "OpenType COLRv1 rendering",
+            "spec_url": "https://docs.microsoft.com/en-us/typography/opentype/spec/colr",
+            "support": {
+              "chrome": {
+                "version_added": "98"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "105",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "gfx.font_rendering.colr_v1.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "OpenType_SBIX": {
+          "__compat": {
+            "description": "OpenType SBIX rendering",
+            "spec_url": "https://docs.microsoft.com/en-us/typography/opentype/spec/sbix",
+            "support": {
+              "chrome": {
+                "version_added": "66"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "9.1"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "OpenType_SVG": {
+          "__compat": {
+            "description": "OpenType SVG rendering",
+            "spec_url": "https://docs.microsoft.com/en-us/typography/opentype/spec/svg",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "31"
```

Comment:
v31 is the earliest I could test - before that my test code gets certificate errors.